<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FormMain</class>
 <widget class="QWidget" name="FormMain">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>768</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="stop">
   <property name="geometry">
    <rect>
     <x>700</x>
     <y>700</y>
     <width>150</width>
     <height>60</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="text">
    <string>停止</string>
   </property>
   <property name="icon">
    <iconset resource="qrc.qrc">
     <normaloff>:/rc/icon/stop.png</normaloff>:/rc/icon/stop.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>32</width>
     <height>32</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="run">
   <property name="geometry">
    <rect>
     <x>860</x>
     <y>700</y>
     <width>150</width>
     <height>60</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="text">
    <string>运行</string>
   </property>
   <property name="icon">
    <iconset resource="qrc.qrc">
     <normaloff>:/rc/icon/run.png</normaloff>:/rc/icon/run.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>32</width>
     <height>32</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="menu">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>700</y>
     <width>150</width>
     <height>60</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="text">
    <string>菜单</string>
   </property>
   <property name="icon">
    <iconset resource="qrc.qrc">
     <normaloff>:/rc/icon/menu.png</normaloff>:/rc/icon/menu.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>32</width>
     <height>32</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="lack">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>700</y>
     <width>200</width>
     <height>60</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="text">
    <string>缺料停机</string>
   </property>
   <property name="icon">
    <iconset resource="qrc.qrc">
     <normaloff>:/rc/icon/lack.png</normaloff>:/rc/icon/lack.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>32</width>
     <height>32</height>
    </size>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="help">
   <property name="geometry">
    <rect>
     <x>980</x>
     <y>5</y>
     <width>40</width>
     <height>40</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="qrc.qrc">
     <normaloff>:/rc/icon/help.png</normaloff>:/rc/icon/help.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>24</width>
     <height>24</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="grade">
   <property name="geometry">
    <rect>
     <x>825</x>
     <y>5</y>
     <width>150</width>
     <height>40</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="text">
    <string>LV3</string>
   </property>
   <property name="icon">
    <iconset resource="qrc.qrc">
     <normaloff>:/rc/icon/LV3.png</normaloff>:/rc/icon/LV3.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>80</width>
     <height>13</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="online">
   <property name="geometry">
    <rect>
     <x>5</x>
     <y>5</y>
     <width>100</width>
     <height>40</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="text">
    <string>联机</string>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="time">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>5</y>
     <width>400</width>
     <height>40</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="text">
    <string>2022-10-17 16:05:23</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QStackedWidget" name="page">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>50</y>
     <width>1024</width>
     <height>590</height>
    </rect>
   </property>
   <widget class="FormMainZero" name="page_Zero"/>
   <widget class="FormMainComb" name="page_Comb"/>
   <widget class="FormMainRec" name="page_Rec"/>
   <widget class="FormMainModifyParam" name="page_ModifyParam"/>
   <widget class="FormMainModifyTime" name="page_ModifyTime"/>
   <widget class="FormMainAlarm" name="page_Alarm"/>
   <widget class="FormCamera" name="page_Camera"/>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>690</y>
     <width>1024</width>
     <height>3</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>640</y>
     <width>1024</width>
     <height>50</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QPushButton" name="pushButton">
      <property name="minimumSize">
       <size>
        <width>128</width>
        <height>50</height>
       </size>
      </property>
      <property name="text">
       <string>置零</string>
      </property>
      <property name="checkable">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_2">
      <property name="minimumSize">
       <size>
        <width>128</width>
        <height>50</height>
       </size>
      </property>
      <property name="text">
       <string>组合</string>
      </property>
      <property name="checkable">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_3">
      <property name="minimumSize">
       <size>
        <width>128</width>
        <height>50</height>
       </size>
      </property>
      <property name="text">
       <string>统计</string>
      </property>
      <property name="checkable">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_4">
      <property name="minimumSize">
       <size>
        <width>128</width>
        <height>50</height>
       </size>
      </property>
      <property name="text">
       <string>参数调整</string>
      </property>
      <property name="checkable">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_5">
      <property name="minimumSize">
       <size>
        <width>128</width>
        <height>50</height>
       </size>
      </property>
      <property name="text">
       <string>时间调整</string>
      </property>
      <property name="checkable">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_6">
      <property name="minimumSize">
       <size>
        <width>128</width>
        <height>50</height>
       </size>
      </property>
      <property name="text">
       <string>故障处理</string>
      </property>
      <property name="checkable">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QPushButton" name="camera">
      <property name="minimumSize">
       <size>
        <width>128</width>
        <height>50</height>
       </size>
      </property>
      <property name="text">
       <string>摄像头</string>
      </property>
      <property name="checkable">
       <bool>true</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="net">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>5</y>
     <width>40</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="wifi">
   <property name="geometry">
    <rect>
     <x>155</x>
     <y>5</y>
     <width>40</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_test">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>700</y>
     <width>100</width>
     <height>60</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>FormMainZero</class>
   <extends>QWidget</extends>
   <header>FormMainZero.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>FormMainComb</class>
   <extends>QWidget</extends>
   <header>FormMainComb.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>FormMainRec</class>
   <extends>QWidget</extends>
   <header>FormMainRec.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>FormMainModifyParam</class>
   <extends>QWidget</extends>
   <header>FormMainModifyParam.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>FormMainModifyTime</class>
   <extends>QWidget</extends>
   <header>FormMainModifyTime.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>FormMainAlarm</class>
   <extends>QWidget</extends>
   <header>FormMainAlarm.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>FormCamera</class>
   <extends>QWidget</extends>
   <header>FormCamera.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>menu</tabstop>
  <tabstop>lack</tabstop>
  <tabstop>stop</tabstop>
  <tabstop>run</tabstop>
  <tabstop>pushButton</tabstop>
  <tabstop>pushButton_2</tabstop>
  <tabstop>pushButton_3</tabstop>
  <tabstop>pushButton_4</tabstop>
  <tabstop>pushButton_5</tabstop>
  <tabstop>pushButton_6</tabstop>
  <tabstop>camera</tabstop>
  <tabstop>online</tabstop>
  <tabstop>grade</tabstop>
  <tabstop>help</tabstop>
 </tabstops>
 <resources>
  <include location="qrc.qrc"/>
 </resources>
 <connections/>
</ui>
